.changeset/moody-aliens-begin.md

```
@@ -0,0 +1,5 @@
+---
+"ensindexer": minor
+---
+
+Made `ponder.schema.ts` to explicitly import just ENSIndexer Schema.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)